### DISCUSSION TITLE
Duplicate Items in Visual Theme Builder

### DISCUSSION THREAD

**Discussion by Code711** [elementor/elementor#25682] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

In the "new" Visual Theme Builder is no option to duplicate items. This ist strange since this option is already available in the old Table View of the Theme Builder?! 

### Describe the solution you'd like.

The three dot menu […] in the Visual Theme Builder should simply contain also the option for "duplicate".

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by brent-e** [comment]
This would be very useful. I frequently need to reuse a given template with some slight modifications, such as when using it for a different CPT. Currently I need to export the template and then import it back. a 'Duplicate' button would be a simpler solution.

**Comment by vgstef** [comment]
In fact, I'd love to see a Duplicate function available for posts and pages too. I often create a few versions of the same page to try different designs or to create a page for a similar content.

**Comment by pumadog** [comment]
I always generate a new one and pick the existing one as the template for it. In the end it's a duplication with 1–2 more clicks.
